snprc_ehr/src/org/labkey/snprc_ehr/table/SNPRC_EHRCustomizer.java

```diff
@@ -77,46 +78,25 @@ public void customize(TableInfo table)
             doTableSpecificCustomizations((AbstractTableInfo) table);
             doCalculatedCustomizations((AbstractTableInfo) table);
             //TODO: customizeColumns((AbstractTableInfo) table);
```

Comment:
Can we also remove this line if we're going to remove the method?

Reply:
Sure. I also didn't remove the date customization for the spf results bleed date. I'll do that and push it at the same time.